src/screens/PrivateChat/ChatList/index.js

```diff
@@ -225,6 +245,10 @@ class ChatListScreen extends NavigatorComponent<Props, State> {
           cancelButtonIndex={newChatOptions.length - 1}
           onPress={this.onChatAction}
         />
+        <MoreMenuModal
+          visible={this.state.showModal === 'more_menu'}
```

Comment:
I see that other identifiers are also using invalid case (we're using lowerCamelCase). Can you please update them? (e.g. 'moreMenu', etc)

Reply:
fixed

Reply:
fixed